dspace-api/src/main/resources/spring/spring-dspace-addon-import-services.xml

```diff
@@ -63,6 +75,42 @@
           </property>
     </bean>
 
+    <bean id="CsvImportService"
+          class="org.dspace.importer.external.csv.service.CharacterSeparatedImportMetadataSourceServiceImpl" scope="singleton">
```

Comment:
Scope singleton is the default, you can leave out the initial scope declaration.